engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2405,6 +2540,46 @@ private LogicalOperatorInfo fillLogicalOperatorInfo(OperatorMeta operator)
     return loi;
   }
 
+  private LogicalModuleInfo fillLogicalModuleInfo(ModuleMeta module, boolean flatten, String parentModuleName)
+  {
+    LogicalModuleInfo lmi = new LogicalModuleInfo();
+    lmi.name = module.getName();
+    lmi.className = module.getModule().getClass().getName();
+    if (flatten) {
+      for (OperatorMeta operatorMeta : getLogicalPlan().getAllOperators()) {
```

Comment:
@tushargosavi : Correct me if I am wrong... 
Let's say I have a module M1 it has operator O1 and inner module M2. M2 has operator O2. After flattening M1 will have operator meta for O1 with moduleName M1 and operator meta for O2 with moduleName M2. Is this correct? If yes then when you iterate over module.getDag().getAllOperator() will the following check not help in filtering out O1 from O2?